docassemble/assemblylinewizard/data/questions/assembly_line.yml

```diff
@@ -614,15 +612,21 @@ code: |
       progress += increment
       logic_list.append("set_progress(" + str(round(progress,2)) +")")    
     logic_list.append(field.trigger_gather)
+    
+    if field.trigger_gather.endswith('.signature'):
+      signatures.add(field.trigger_gather)
   
-  # Save a snapshot of interview answers. 
-  # We only want a few anonymous variables
-  logic_list.append("# Save (anonymized) interview statistics.")
-  logic_list.append("store_variables_snapshot(data={'zip': users[0].address.zip})")
+  # Build the mandatory code block that will control overall flow
+  # Need this below collecting signatures
+  main_code = interview.questions.appendObject() # 'main code block'
+  main_code.type = 'main order'
+  main_code.intro = 'al_intro_screen'  # Splash page for your organization
```

Comment:
Is this in here on purpose so it's easier for forkers to find and change it? I was going to move it into the .py, but I wasn't sure.

Reply:
No strong reason either way